docs/installation/advanced-installation-options.rst

```diff
@@ -51,3 +51,37 @@ install the newer version of ``presto-admin`` by following the instructions in
 the installation section (see :ref:`presto-admin-installation-label`). Your
 configuration files located in ``/etc/opt/prestoadmin`` will remain intact and
 continue to be used by the newer version of ``presto-admin``.
+
+Coordinator failover
+--------------------
+Presto does not yet support automatic failover for the coordinator. You can
+migrate to a new coordinator using the ``presto-admin`` -H and -x flags
+to include and exclude hosts in your command, respectively.
+
+To view these ``presto-admin`` options, use the ``--extended-help`` flag.
+
+You can switch to a new coordinator by following the steps below:
+
+1. Shutdown the cluster.
```

Comment:
=> Stop Presto on all the nodes where it is running by running sudo ./presto-admin server stop.